health-integrations/README.md

```diff
@@ -14,3 +15,27 @@ InjectorBuilder
     )
     .createInjector();
 ```
+
+### Archaius2-backed IndicatorMatcher
+Filtering of which HealthIndicators will be used in communicating the health status of the application is provided via an [Archaius2](https://github.com/Netflix/archaius/tree/2.x) implementation of [IndicatorMatcher](https://github.com/Netflix/runtime-health/tree/health-api/src/main/java/com/netflix/runtime/health/api/IndicatorMatcher.java).
+
+```
+InjectorBuilder
+    .fromModules(
+        // ... other modules ...
+        new HealthModule(),
+        new EurekaHealthStatusBridgeModule(),
+        new ArchaiusHealthStatusFilterModule(),
+        // ... other modules ...
+    )
+    .createInjector();
+```
+
+Indicators can now be included/excluded by setting the following properties in Archaius2. Excluded indicators will take priority over included ones.
+```
+health.status.indicators.include=com.myproject.MyIndicator
```

Comment:
Let's make this example with multiple included indicators so that how to do this for a list of indicators can be easily understood.